Remotion/Web/Development.WebTesting.IntegrationTests/TreeViewControlObjectTest.cs

```diff
@@ -95,7 +96,8 @@ public void TestNodeIsChecked ()
       Assert.That (
           () => rootNode.IsChecked(),
           Throws.InstanceOf<WebTestException>()
-              .With.Message.EqualTo ("The checkbox could not be found: Unable to find xpath: ./tbody/tr/td[a[contains(@onclick, 'TreeView_SelectNode')]]/input[@type='checkbox']"));
+              .With.Message.EqualTo (

```

Comment:
I'm missing the consistency in the formatting. this would be the preferred style for lines we have reformatted in this PR and the message does not fit in the With.Message.EqualTo line.